### PR TITLE
MM-18870 Workflow examples.

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: source/samples/workflow_multiple.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
+[
+    {
+        "name": "HighSeverity",
+        "triggers": [
+            {
+                "name": "ServiceDown",
+                "type": "text",
+                "match": "DOWN",
+                "team_name": "rrrr",
```
Comment (jasonblais) [inline source/samples/workflow_multiple.json]:
```suggestion
                "team_name": "infosec",
```

File: source/samples/workflow_multiple.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        ]
+    },
+    {
+        "name": "MediumSevertiy",
+        "triggers": [
+            {
+                "name": "ServiceCritical",
+                "type": "text",
+                "match_regex": "(CRITICAL.*SERVICE|SERVICE.*CRITICAL)",
+                "team_name": "rrrr",
```
Comment (jasonblais) [inline source/samples/workflow_multiple.json]:
```suggestion
                "team_name": "infosec",
```

File: source/samples/workflow_multiple.json [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        ]
+    },
+    {
+        "name": "Warning",
+        "triggers": [
+            {
+                "name": "ServiceWarning",
+                "type": "text",
+                "match": "WARNING",
+                "team_name": "rrrr",
```
Comment (jasonblais) [inline source/samples/workflow_multiple.json]:
```suggestion
                "team_name": "infosec",
```

File: source/samples/workflow_multiple.json [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+            }
+        ],
+        "steps": [
+            {
+                "name": "SendAlert",
+                "start_actions": [
+                    {
+                        "name": "IntroPost",
+                        "type": "post",
+                        "team_name": "rrrr",
```
Comment (jasonblais) [inline source/samples/workflow_multiple.json]:
```suggestion
                        "team_name": "infosec",
```

**Comment by crspeller** [comment]
@jasonblais Applied team name. What do you mean by cross-workflow vars?

**Comment by jasonblais** [comment]
@crspeller 

> What do you mean by cross-workflow vars?

I meant having workflow #1 use a variable from workflow #2. 

I saw it in the title of https://mattermost.atlassian.net/browse/MM-18870, thought it would mean that, might be wrong though.

**Comment by crspeller** [comment]
@jasonblais That's not possible.

**Comment by jasonblais** [comment]
@crspeller Ah okay, will approve and merge

**Pull request merged**